### DISCUSSION TITLE
"simple_storage_file is not defined at 3:45

### DISCUSSION THREAD

**Discussion by benjamin15790** [smartcontractkit/full-blockchain-solidity-course-py#896] [Q&A] [answered]
Hi i am getting an error with "simple_storage_file is not defined.

This is the section of code which i believe i have written correctly. But maybe this is a pathway issue, which i don't understand how to setup pathways. 

Any assistance would be greatly appreciated. 

```Python
compiled_sol = compile_standard(
        {
            "language": "Solidity",
            "sources": {"SimpleStorage.sol": {"content": simple_storage_file}},
            "settings": {
                "outputSelection": {
                    "*": {"*": ["abi", "metadata", "evm.bytecode", "evm.sourceMap"]}
                }
            },
        },
        solc_version="0.6.0",
    )
    print(compiled_sol)
```

**Comment by cromewar** [comment]
Hello @benjamin15790 do you have this code above?

```Python
with open("./SimpleStorage.sol", "r") as file:
    simple_storage_file = file.read()
```

**Comment by n4n0b1t3** [comment, marked as answer]
Hi Benjamin,
videos of this kind of rapidly developing technology tend to be (slightly) outdated before they are uploaded. I would suggest always first referring to the corresponding GitHub repo in this case: https://github.com/PatrickAlphaC/web3_py_simple_storage
For all the following sections of this course please 
1. go to the table of contents https://github.com/smartcontractkit/full-blockchain-solidity-course-py#table-of-contents
2. go to the top of the section you are currently in and
3. go to the GitHub page where you will find the up-to-date source code
Good luck and have fun
nanobite

**Reply by cromewar** [reply]
Nice hint @n4n0b1t3

**Reply by n4n0b1t3** [reply]
What would be your estimation: How many posts could be answered like this?

**Reply by cromewar** [reply]
Dunno depends a lot on the problem itself, but you are right checking the repo code is always something desirable as you code might be updated.

**Comment by n4n0b1t3** [comment]
@benjamin15790 Did this solve your problem? If so, please mark the correct answer. This way others can better find the solution if they run into the same problem.
